examples/systemcalls/systemcalls.c

```diff
@@ -93,7 +158,64 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+    int fd = open(outputfile, O_WRONLY|O_TRUNC|O_CREAT, 0644);
```

Comment:
@Ruthvik-Chanda think about a way you could design this function such that you don't need to duplicate code with the implementation above.